contents/product-engineers/pmf-surveys.md

```diff
@@ -0,0 +1,166 @@
+---
+date: 2023-11-29
+title: "How to drive growth with product-market fit surveys"
+author:
+  - lior-neu-ner
+featuredImage: ../images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+tags:
+  - Product engineers
+  - User research
+---
+
+*"How would you feel if you could no longer use our product?"* a.k.a the product-market fit survey.
+
+This is perhaps the most well-known question in SaaS circles. [Superhuman](https://superhuman.com) famously used it to catapult themselves to over 50,000 paying customers and a whopping **$20 million** in revenue. 
+
+You might be wondering, "How can I do the same?"
+
+In this post we show you just that. We break down [Superhuman's product-market fit engine](https://review.firstround.com/how-superhuman-built-an-engine-to-find-product-market-fit) into an easy step-by-step guide you can follow as well as dive into two case studies.
+
+## A very brief introduction to PMF surveys
+
+> If you're already familiar with product-market fit surveys, skip to the [next section](#how-to-drive-product-market-fit-with-pmf-surveys).
+
+Created by entrepreneur Sean Ellis, the core question (among others) it asks is:
+
+"How would you feel if you could no longer use [product name]?"
+
+- a) Very disappointed
+- b) Somewhat disappointed
+- c) Not disappointed
+
+Your PMF score is the percentage of total responses that are "very disappointed".
+
+For example, let's say you survey `100` people. `50` people respond with "very disappointed", your PMF score would be '50 / 100 = 50%`.
+
+Based on his research of 100+ startups, Ellis believes 40% answering "very disappointed" is a strong signal of product-market fit.
+
+![How to calculate PMF score](../images/blog/pmf-surveys/pmf-score.png)
+
+## How to drive product-market fit with PMF surveys
+
+### Step 1: Survey informed users
+
+The first step is to learn more about your customers and what they think of your app. Do this by creating a [survey](/surveys) with the following questions:
+
+1. How would you feel if you could no longer use our product?
+   
+2. What type of people do you think would most benefit from it?
+
+3. What is the main benefit you receive from it?
+
+4. How can we improve our app for you?
```

Comment:
I don't like this question as it forces users to think of who beyond themself would benefit from the app, basically asking them to do market research for you, which I don't think works well.

I would drop this one.
```suggestion
1. How would you feel if you could no longer use our product?

2. What is the main benefit you receive from it?

3. How can we improve our app for you?
```